templates/ecommerce/checkout.php

```diff
@@ -54,6 +54,7 @@
 				if ( file_exists( $file ) ) {
 					include $file;
 				}
+				$is_zero_price = empty( $checkout_data->total_price );
```

Comment:
On country state change or coupon add remove, only checkout-details.php is re-rendered. In checkout.php $checkout->total_price will not be update.